tuned_lens/nn/lenses.py

```diff
@@ -316,3 +316,66 @@ def forward(self, h: th.Tensor, idx: int) -> th.Tensor:
     def __len__(self) -> int:
         """Return the number of layer translators in the lens."""
         return len(self.layer_translators)
+
+    @th.inference_mode()
```

Comment:
Why isn't this part already in `main`? Maybe you should merge main into this branch before requesting review again.